src/v3/src/components/Widget/index.tsx

```diff
@@ -101,13 +101,18 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
   const [loginHint, setloginHint] = useState<string | null>(null);
   const brandedTheme = mapMuiThemeFromBrand(brandColors, muiThemeOverrides);
 
-  useEffect(() => {
-    // If we need to load a language (or apply custom i18n overrides), do
-    // this now and re-run render after it's finished.
-    if (!Bundles.isLoaded(getLanguageCode(widgetProps))) {
-      (async () => {
-        await loadLanguage(widgetProps);
-      })();
+  // on unmount, remove the language
+  useEffect(() => () => {
+    if (Bundles.isLoaded(getLanguageCode(widgetProps))) {
+      Bundles.remove();
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
On unmount, need to clear language so we can load again (if options have changed).